src/mac/nrn_notarize.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -e
+# App specific password used to request notarization
+password_path="$HOME/.ssh/notarization-password"
+if test -f "$password_path" ; then
+  app_specific_password=`cat "$password_path"`
+else
+  echo "\"$password_path\" does not exist"
+  exit 1
+fi
+
+pkg="$1"
+pkgname="$2"
+
+echo "Notarize request"
+xcrun altool --notarize-app \
+  --primary-bundle-id "edu.yale.neuron.pkg.$pkgname" \
+  --username "michael.hines@yale.edu" \
+  --password "$app_specific_password" \
+  --file "$pkg"
+
+# you should get
+# 2021-01-31 17:43:58.537 altool[3021:3231297] CFURLRequestSetHTTPCookieStorageAcceptPolicy_block_invoke: no longer implemented and should not be called
+# No errors uploading '/Users/michaelhines/neuron/notarize/build/src/mac/build/NEURON-8.0.pkg'.
+# RequestUUID = dc9dd4dd-a942-475a-ab59-4996f938d606
+
+# and eventually get an email in just a few minutes saying that
```

Comment:
If it is just a few minutes, we could implement a sort of loop and check the status and then staple. Like in the `Example` gif here: https://phoenix.yizimg.com/mitchellh/gon